habitat_baselines/train_ppo.py

```diff
@@ -158,7 +158,7 @@ def construct_envs(args):
     return envs
 
 
-def main():
+def run_training():
```

Comment:
```suggestion
def run_training(args):
```
Maybe, lets add `args` as params that will make possible to run this function from tests.

Reply:
I'll leave this as is for now, will make changes like this in a separate PR about making train_ppo testable.